config.yaml

```diff
@@ -97,6 +104,15 @@ graph:
     path: /inputs
   outputs:
   - path: /outputs
+- id: convert-table-to-csv-node
+  job_id: convert-table-to-csv-job
+  inputs:
+  - node_id: metadata-node
+    predicate: '.*'
```

Comment:
If you're not predicating, then you don't need to be connected to the metadata node. Remove this input to run in parallel.